app/model/schema/token.py

```diff
@@ -76,6 +83,27 @@ class ListAllTransferHistoryQuery:
         Optional[TransferSourceEvent], Query(description="source event of transfer")
     ] = None
     data: Annotated[Optional[str], Query(description="source event data")] = None
+    transaction_hash: Annotated[
+        Optional[str], Query(description="transaction hash")
+    ] = None
+    value: Annotated[Optional[int], Query(description="value")] = None
+    value_operator: Annotated[
+        Optional[ValueOperator],
+        Query(
+            description="value filter condition(0: equal, 1: greater than, 2: less than)"
+        ),
+    ] = ValueOperator.EQUAL
```

Comment:
Added the parameters of the filters added to the **Search** API that also can be used in GET requests to the **List** API.
(Filters which expect `datetime` type is not added because it cannot be used in GET.)